src/containers/topbutton/Top.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
Don't include this component inside container, instead put it in components. Also the name `topbutton` seems ambiguous. Please change the name of the folder to `goTopButton` and the file names to `GoTopButton.js` and `GoTopButton.css`.